File: examples/typescript/operations/map/map-signature.ts
// Named overload
map<TInput, TOutput>(
  name: string | undefined,
  items: TInput[],
  mapFunc: MapFunc<TInput, TOutput>,
  config?: MapConfig<TInput, TOutput>,
): DurablePromise<BatchResult<TOutput>>

// Unnamed overload
map<TInput, TOutput>(
  items: TInput[],
  mapFunc: MapFunc<TInput, TOutput>,
  config?: MapConfig<TInput, TOutput>,
): DurablePromise<BatchResult<TOutput>>

File: examples/csharp/operations/map/map-signature.cs
Task<IBatchResult<TResult>> MapAsync<TItem, TResult>(
    IReadOnlyList<TItem> items,
    Func<IDurableContext, TItem, int, IReadOnlyList<TItem>, CancellationToken, Task<TResult>> func,
    string? name = null,
    MapConfig? config = null,
    CancellationToken cancellationToken = default);
